tests/unit/src/test/scala/tests/OverridesSuite.scala

```diff
@@ -17,29 +18,21 @@ class OverridesSuite extends FunSuite with TempDirectories {
       options: TestOptions,
       source: String,
       extractSymbol: String,
-      expectedSymbols: List[String],
-      expectedCount: Int,
-      qualifiedClassName: String = "example.Parent"
+      expectedSymbols: String *,
```

Comment:
It's best to keep this as a single string and pass in multiline strings instead.
```suggestion
       expectedSymbols: String,
```
The motivation for this is that test failures include a multiline formatted string that you can copy-paste back into the source code. If we keep the vararg parameter then you need to manually write out the new expected output or manually split up the failure message somehow.

Reply:
The output is identical, and in both situations theres some manual work required to fix the test with the right expected result, so Im not sure I follow on what the advantage is youre describing. I could see it being better from a visual clarity perspective though, so Ill change it anyways

New:
```scala
checkOverrides(
	// ...
	"""example/Parent#toString().
	|java/lang/Object#toStrig().
	|""".stripMargin
)
```
```diff
==> X tests.OverridesSuite.multiple  0.071s munit.ComparisonFailException: /home/noah/Sourcegraph/lsif-java/tests/unit/src/test/scala/tests/OverridesSuite.scala:34
33:        .getOverriddenSymbolsList.stream.collect(Collectors.joining("\n"))
34:      assertNoDiff(expectedSymbols, syms)
35:    }
diff assertion failed
=> Obtained
    """|example/Parent#toString().
       |java/lang/Object#toStrig().
       |""".stripMargin
=> Diff (- obtained, + expected)
 example/Parent#toString().
-java/lang/Object#toStrig().
+java/lang/Object#toString().
    at munit.FunSuite.assertNoDiff(FunSuite.scala:11)
    at tests.OverridesSuite.$anonfun$checkOverrides$1(OverridesSuite.scala:34)
```

Current:
```scala
checkOverrides(
	// ...
	"example/Parent#toString().",
	"java/lang/Object#toStrig().",
)
```
```diff
==> X tests.OverridesSuite.multiple  0.053s munit.ComparisonFailException: /home/noah/Sourcegraph/lsif-java/tests/unit/src/test/scala/tests/OverridesSuite.scala:35
34:        .getOverriddenSymbolsList.stream.collect(Collectors.joining("\n"))
35:      assertNoDiff(expectedSyms, syms)
36:    }
diff assertion failed
=> Obtained
    """|example/Parent#toString().
       |java/lang/Object#toStrig().
       |""".stripMargin
=> Diff (- obtained, + expected)
 example/Parent#toString().
-java/lang/Object#toStrig().
+java/lang/Object#toString().
    at munit.FunSuite.assertNoDiff(FunSuite.scala:11)
    at tests.OverridesSuite.$anonfun$checkOverrides$1(OverridesSuite.scala:35)
```

Reply:
The output from a failing test remains identical but the call-site can be updated with a single copy-paste if you use the multiline string syntax. If you use varargs syntax at call-site then you have to post-process the error message.

Reply:
Not sure Im seeing that in this example :thinking: If multiple lines failed I still need to add `|`/remove `+` or `-`, unless Im missing something here :sweat_smile: 
Either way, code's been updated :slightly_smiling_face:

Reply:
You can copy-paste the "Obtained" section above the "Diff" section

Reply:
It all makes sense now, I had passed the params the wrong way around to `assertNoDiff` :facepalm:

Reply:
Gharg! I didn't check. MUnit actually uses the opposite order for actual/expected compared to JUnit (some people are upset about it). The reason the obtained value comes first is because it's prettier to have the multiline string appear as the last argument.